scripts/tests/backend-test.sh

```diff
@@ -95,14 +95,14 @@ function run_tests() {
   fi
 
   # Switch to esplora and run wallet tests
-  export FM_BITCOIN_RPC_KIND="esplora"
-  export FM_BITCOIN_RPC_URL="http://127.0.0.1:$FM_PORT_ESPLORA"
+  export FM_TEST_BACKEND_BITCOIN_RPC_KIND="esplora"
+  export FM_TEST_BACKEND_BITCOIN_RPC_URL="http://127.0.0.1:$FM_PORT_ESPLORA"
 
   if [ -z "${FM_TEST_ONLY:-}" ] || [ "${FM_TEST_ONLY:-}" = "esplora" ]; then
     >&2 echo "### Testing against esplora"
     cargo nextest run --locked --workspace --all-targets \
       ${CARGO_PROFILE:+--cargo-profile ${CARGO_PROFILE}} ${CARGO_PROFILE:+--profile ${CARGO_PROFILE}} \
-      ${TEST_ARGS_THREADED} \
+      ${TEST_ARGS_SERIALIZED} \
```

Comment:
Intentional. I'm not sure why, but some of the esplora tests can take a long time and since we lock bitcoind we can exceed the 120 second timeout if running all tests in parallel.